docs/content/concepts/resources.mdx

```diff
@@ -5,7 +5,24 @@ description: Resources enable you to separate graph logic from environment, and
 
 # Resources
 
-Resources provide a way to manage dependencies to external components and share implementations across multiple ops in a job.
+Resources provide a way to manage dependencies to external components and share implementations across multiple [ops](/concepts/ops-jobs-graphs/ops) or [assets](/concepts/assets/software-defined-assets) in a [job](/concepts/ops-jobs-graphs/jobs-graphs).
```

Comment:
Not sure if this matters enough to be worth fixing, but assets can have resources even when the assets are not included in jobs.